velox/functions/prestosql/tests/URLFunctionsTest.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gmock/gmock.h>
+
+#include "velox/functions/prestosql/tests/FunctionBaseTest.h"
+
+namespace facebook::velox {
+
+namespace {
+class URLFunctionsTest : public functions::test::FunctionBaseTest {};
```

Comment:
There is a lot of repetition in this test. Check out testUrlExtract and testUrlExtractParameter in presto-main/src/test/java/com/facebook/presto/operator/scalar/TestUrlFunctions.java to see if the approach taken there would work here as well.